consts.js

```diff
@@ -4,6 +4,9 @@ const MAX_ATTEMPTS = 6;
 const GAME_STATE_KEYS = ['gameID', 'category', 'failsCounter', 'encodedPhrase', 'endState'];
 const LETTER_STATUS_CORRECT = 'LETTER_STATUS_CORRECT';
 const LETTER_STATUS_INCORRECT = 'LETTER_STATUS_INCORRECT';
+const QWERTY = 'QWERTY';
+const ALPHABETICALLY = 'ALPHABETICALLY';
```

Comment:
You could put those 2 variables into:
const KeyboardType = {
 QWERTY: 'QWERTY',
 ALPHABETICAL: 'ALPHABETICAL'
}
and then put KeyboardType into export